torchao/_models/llama/perf_profile.py

```diff
@@ -0,0 +1,442 @@
+"""
```

Comment:
This script is still quite similar to https://github.com/pytorch/ao/blob/main/torchao/_models/llama/generate.py and was hoping we could converge the two